cfgov/jinja2/v1/ask-cfpb/answer-page.html

```diff
@@ -1,71 +1,125 @@
 {% extends 'layout-2-1-bleedbar.html' %}
+{% import 'macros/time.html' as time %}
+{% import 'tags.html' as tags with context %}
 {% import 'form_block.html' as form_block with context %}
 
+{% block title %}
+    {{ page.question|striptags }}
+{% endblock %}
+
+{% block desc %}
+    {% if page.snippet %}
+        {{ page.snippet|striptags|safe}}
+    {% elif page.answer %}
+        {{ page.answer[:500]|striptags|safe}}
```

Comment:
Why no handle it within python? You are just limiting the answer to the first 500 chars, correct?